index.html

```diff
@@ -0,0 +1,289 @@
+                  statements, and verification of non-filing letters.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md">
+            <div class="card bg-light bg-opacity-10">
+              <div class="card-body align-items-center">
+                <img src="./assets/images/tools/tools3.png" alt="tools 3" />
+                <h5 class="card-title orangeColor">Amend Return</h5>
+                <p class="card-text">
+                  Check the status of your Form 1040-X, Amended U.S. Individual
+                  Income Tax Return, for this year and up to three prior years.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md">
+            <div class="card bg-light bg-opacity-10">
+              <div class="card-body align-items-center">
+                <img src="./assets/images/tools/tools4.png" alt="tools 4" />
+                <h5 class="card-title orangeColor">Refund</h5>
+                <p class="card-text">
+                  Check your income status tax refund. If you e-file, allow 36
+                  hours for your information to appear and 3 weeks to a refund.
+                </p>
+              </div>
+            </div>
+          </div>
+          <div class="col-md">
+            <div class="card bg-light bg-opacity-10">
+              <div class="card-body align-items-center">
+                <img src="./assets/images/tools/tools6.png" alt="tools 5" />
+                <h5 class="card-title orangeColor">Payment</h5>
+                <p class="card-text">
+                  Using 3 identification safeguards your privacy and secures
+                  your information when making your tax payment.
+                </p>
+              </div>
+            </div>
+          </div>
+        </div>
+
+        <a href="#" class="text-decoration-underline whiteColor findmore fs-4"
+          >Find out more</a
+        >
+        <a class="btn orangeBgd whiteColor w-75 py-3" href="#" role="button"
+          >Find out more</a
+        >
+      </div>
+    </section>
+
+    <!-- courses section -->
+    <section class="courseSection text-center">
+      <div class="container-fluid width8 py-5">
+        <h2>Training and Certification</h2>
+        <div class="linesmall orangeColor orangeBgd mx-auto"></div>
+        <div class="row py-5 cousesContainer">
+          <div class="col-md-6 cousesContainer1"></div>
+          <div class="col-md-6 cousesContainer2"></div>
+        </div>
+        <button class="btn w-100 btn-outline-secondary moreCourses">
+          <b>MORE </b><span class="orangeColor px-2"> <b>V</b> </span>
+        </button>
+      </div>
+    </section>
+    <!-- partners section -->
+    <section class="partners darkBgd2 text-light homedesk">
+      <div class="container-fluid width8 py-5 text-center">
+        <h2>Partners</h2>
```

Comment:
- [ ] :thinking: I think it would be better to underline this title, and give it a darker color as per the Behance design as shown below. :recycle: 


- Your design:
![Ypartners](https://user-images.githubusercontent.com/97834160/210766244-97c6fb4c-ac8e-42f9-b96c-3b1b71f0f86f.png)

- Behance design:

![Bpartner](https://user-images.githubusercontent.com/97834160/210766362-e37cf8ff-1f11-4862-bea2-00609c258814.png)